### PR TITLE
Fix bake init bug, update getting started

### PR DESCRIPTION
**Pull request by ripclawffb**
When setting this up for the first time on my machine, I got an error because I didn't have an `EDITOR` environment variable set. This change fixes those errors.

```sh
$ bake init
/root/bin/bake: line 543: EDITOR: unbound variable
```

Also, when running `bake get-started` I got a similar error.

```sh
$ bake get-started
/root/Bakefile: line 7: EDITOR: unbound variable
```

I also updated the README.md because copying and pasting the commands from the examples didn't work. These updates should make it easier to copy and paste to run through the library tutorial.

Changed the default editor to `vi` since not all environments have `vim` installed.

### CODE REVIEW

File: bake [hunk 1]
```diff
@@ -540,18 +540,17 @@ function bake_update () {
...
 
 # bake_require github.com/kyleburton/bake-recipes/docker/docker.sh
 
+# Set default editor if not defined
+export EDITOR="${EDITOR:-vi}"
+
 bake_task get-started "Get started by editing this Bakefile"
 function get-started () {
-  if [ -n "$EDITOR" ]; then
     "$EDITOR" Bakefile
```
Comment (kyleburton) [inline bake:553]: would you kindly include an else that echo's a message?  the task description indicates that it will bring up an editor.

**Comment by kyleburton** [comment]
This is awesome, ty!  I only have one small request (above), and will ask if you've run shellcheck on the changes?

File: bake (hunk 1 above)
Comment (ripclawffb) [reply bake:553]: I set the default to `vi` in line 549, so there won't be an instance where it's not set.

```
export EDITOR="${EDITOR:-vi}"
```

**Comment by ripclawffb** [comment]
> This is awesome, ty! I only have one small request (above), and will ask if you've run shellcheck on the changes?

Thanks! :) I've made the `shellcheck` updates as requested.

File: bake (hunk 1 above)
Comment (kyleburton) [reply bake:553]: check, ty!

**Review by kyleburton — APPROVED** [review]

**Comment by kyleburton** [comment]
Almost forgot, tho this PR is closed, would you mind adding yourself as a contributor? (in the readme) a contact email is optional, though I'd like for the folks who invested in this tool and project to have that investment noted for all to see :-D

**Pull request merged**